demo/CMakeLists.txt

```diff
@@ -22,7 +22,10 @@ add_subdirectory(sdk)
 
 # add static demo
 add_executable(demostatic main.cc chainmain.cc cronos.cc extra.cc third_party/easywsclient/easywsclient.cpp)
-if (WIN32)
+if (WIN32 AND MSYS)
+  target_link_libraries(demostatic PUBLIC ${PLAY_CPP_SDK_LIB} ${RUST_LIB} Ncrypt userenv ntdll Secur32 crypt32 ws2_32 ntdll)
```

Comment:
RUST_LIB is unnecessary. It is already link into PLAY_CPP_SDK_LIB by https://github.com/cronos-labs/play-cpp-sdk/blob/main/demo/sdk/CMakeLists.txt#L22

PLAY_CPP_SDK_LIB is a output of SDK/CMakeLists.txt.

Reply:
ok, i'll check